compose-app/src/main/java/com/mapbox/maps/extension/compose/ComposeUtils.kt

```diff
@@ -0,0 +1,98 @@
+package com.mapbox.maps.extension.compose
```

Comment:
```suggestion
package com.mapbox.maps.compose
```

and maybe rename file to `MapboxCompose`?